package.json

```diff
@@ -24,6 +25,7 @@
     "chai": "3.3.0",
     "grunt": "0.4.5",
     "grunt-eslint": "17.1.0",
+    "grunt-jscs": "^2.1.0",
```

Comment:
Good call. I installed this from the CLI and I've been bit by `^` before. And we have greenkeeper anyway, so should be good.